### PR TITLE
Restyling

### LINKED ISSUE #46: Form Restyle
- Implement the form restyle as seen [here](https://www.figma.com/file/UnjR8tbWQ49DHxHbtLmP7V/Full-Redesign-(All Breakpoints)?node-id=0%3A1). 
- Mobile view is not necessary for this ticket. 
- Consider making a 'breakpoint' (# of characters) at which we switch from the text-in-button Radio styling to the text-beside-button Radio styling.
- Ask designers if you have questions!!!
- Use styled-components for css rules pls 🙏 - feel free to use MUI components but be warned it might make styling more difficult

Specs:
- [x] Styled radio text-in-button
- [x] Styled radio text-beside-button
- [x] Styled text input
- [x] Styled continue
- [x] Styled next/back
- [x] Styled section header
- [x] Placeholder left section tracking bar (populate with dummy values)
- [x] Styled Nav Bar
- [x] Styled quote blocks
  - [x] Parse out anything in the `question` field that is contained in double quotes
  - [x] pic: Screen Shot 2022-04-15 at 3 18 13 AM
  - [x] [link](https://www.figma.com/file/UnjR8tbWQ49DHxHbtLmP7V/Full-Redesign-(All-Breakpoints)?node-id=2%3A668)

### PR DESCRIPTION
**Pull request by imanmoreira**
Closes #46

### CODE REVIEW

File: components/DynamicForm/MyInput.tsx [hunk 1]
```diff
@@ -1,10 +1,8 @@
 import { FieldHookConfig, useField } from 'formik'
 import React, { ChangeEvent } from 'react'
-import { FormAnswer, TextFormAnswer } from '../../utils/FormContext'
-import { QuestionText } from '../FormStyles/QuestionText'
+import { TextFormAnswer } from '../../utils/FormContext'
```
Comment (imanmoreira) [inline components/DynamicForm/MyInput.tsx:3]: Only removing dependencies and changing the string width/height to a number

File: components/DynamicForm/MyContinue.tsx [hunk 2]
```diff
@@ -14,7 +13,7 @@ Represents a "Continue" question in the form, which just displays some informati
 const MyContinue: React.FC<ContinueProps> = ({ label, onMount }) => {
   useEffect(() => {
     onMount()
-  }, [])
+  }, [onMount])
```
Comment (imanmoreira) [inline components/DynamicForm/MyContinue.tsx:16]: Only removing dependencies and dealing with react warnings

File: components/Critical/NavBar.tsx [hunk 3]
```diff
@@ -1,67 +1,55 @@
...
-            variant="middle"
-            classes={{ root: classes.divider }}
-          />
-          <Typography variant="body2" classes={{ root: classes.typography }}>
-            edlawproject@publiccounsel.net
-          </Typography>
-        </div>
+    <AppBar
+      position="static"
+      style={{
```
Comment (imanmoreira) [inline components/Critical/NavBar.tsx:35]: moved a lot of the styling to inline -- there were some issues in making them styled components but they didn't feel big enough to make it seem worth it to not do it inline

File: components/DynamicForm/MyResult.tsx [hunk 4]
```diff
@@ -7,12 +7,16 @@ import {
...
 } from '../../utils/FormContext'
 import { Question } from '../../models'
-import { QuestionText } from '../FormStyles/QuestionText'
+import { QuestionsWithBlockText, AnswerText } from '../FormStyles/QuestionText'
 import styled from 'styled-components'
 import { StyledTextInput } from '../FormStyles/InputBox'
 import { QuestionType } from '../../models/question'
 import QuestionLayout from '../FormStyles/QuestionLayout'
 
+const SingleQuestionResponseDiv = styled.div`
```
Comment (imanmoreira) [inline components/DynamicForm/MyResult.tsx:16]: Just a rename for horizontalDiv + removing unnecessary styling

File: components/FormStyles/Button.tsx [hunk 5]
```diff
@@ -1,23 +1,22 @@
 import styled, { css } from 'styled-components'
+import { COLORS } from '../../constants/colors'
 
 interface buttonProps {
   primary?: boolean
 }
 
 export const Button = styled.button`
```
Comment (imanmoreira) [inline components/FormStyles/Button.tsx:8]: moving unused styling pieces + making changes

File: components/FormStyles/ContactInfo.tsx [hunk 6]
```diff
@@ -133,58 +133,58 @@ const ContactFormik: React.FC = () => {
             <ContactContent>
               <ContactColumn>
                 <StyledTextInput
-                  width="330px"
-                  height="53px"
+                  width={330}
```
Comment (imanmoreira) [inline components/FormStyles/ContactInfo.tsx:136]: ignore this file

File: components/FormStyles/InputBox.tsx [hunk 7]
```diff
@@ -1,21 +1,28 @@
 import styled from 'styled-components'
 import { FieldHookConfig, useField } from 'formik'
 import React, { ChangeEvent } from 'react'
+import { COLORS } from '../../constants/colors'
 
 interface InputBoxProps {
-  width: string
-  height: string
+  width: number
```
Comment (imanmoreira) [inline components/FormStyles/InputBox.tsx:7]: most of this file is just converting the width/height to be a string aside from the syling in the inputBox below

File: components/FormStyles/PasswordInputBox.tsx [hunk 8]
```diff
@@ -14,8 +14,8 @@ export const IconButton = styled.button`
 `
 
 interface PasswordProps {
-  width: string
-  height: string
+  width: number
```
Comment (imanmoreira) [inline components/FormStyles/PasswordInputBox.tsx:17]: can ignore

File: components/FormStyles/QuestionText.tsx [hunk 9]
```diff
@@ -2,10 +2,51 @@ import styled from 'styled-components'
...
+const BlockQuote: React.FC<QuestionTextProps> = ({ questionText }) => {
+  return (
+    <div style={{ display: 'flex', padding: '10px, 0px, 10px, 0px' }}>
+      <QuoteBlockBar />
+      <QuestionText>{questionText}</QuestionText>
+    </div>
+  )
+}
+
+export const QuestionsWithBlockText: React.FC<QuestionTextProps> = ({
```
Comment (imanmoreira) [inline components/FormStyles/QuestionText.tsx:35]: logic here is to essentially break up any questions with quotes in them, and the quotes should be displayed, so every odd numbered section of the split should be between two quotes

File: components/Login/ForgotPassword.tsx [hunk 10]
```diff
@@ -46,8 +46,8 @@ function ForgotPassword() {
           </SubTitle>
           <TextButtonDiv>
             <StyledTextInput
-              width="600px"
-              height="53px"
+              width={600}
```
Comment (imanmoreira) [inline components/Login/ForgotPassword.tsx:49]: ignore

File: components/Login/SignIn.tsx [hunk 11]
```diff
@@ -63,16 +63,16 @@ function SignIn() {
           <SubTitle>Please sign in.</SubTitle>
           <SubContainer>
             <StyledTextInput
-              width="600px"
-              height="53px"
+              width={600}
```
Comment (imanmoreira) [inline components/Login/SignIn.tsx:66]: ignore

File: components/Login/SignUp.tsx [hunk 12]
```diff
@@ -64,8 +64,8 @@ function Signup() {
           <SubTitle>Let us get your account up and running.</SubTitle>
           <div>
             <StyledTextInput
-              width="330px"
-              height="53px"
+              width={330}
```
Comment (imanmoreira) [inline components/Login/SignUp.tsx:67]: ignore

File: components/Critical/NavBar.tsx [hunk 13]
```diff
@@ -1,67 +1,55 @@
...
-    flex: 'none',
-  },
-}))
+const NeedHelpContainer = styled.div`
+  display: flex;
+  margin-right: 120px;
+  align-items: center;
+`
 
+const StuledHelpIcon = styled.div`
```
Comment (dillydally414) [inline components/Critical/NavBar.tsx:16]:
```suggestion
const StyledHelpIcon = styled.div`
```

File: constants/colors.ts [hunk 14]
```diff
@@ -0,0 +1,7 @@
+export const COLORS = {
```
Comment (dillydally414) [inline constants/colors.ts:1]: smart asf

File: components/FormStyles/QuestionText.tsx [hunk 15]
```diff
@@ -2,10 +2,50 @@ import styled from 'styled-components'
...
+  margin-right: 12px;
 `
+
+interface QuestionTextProps {
+  questionText: string
+}
+
+const BlockQuote: React.FC<QuestionTextProps> = ({ questionText }) => {
+  return (
+    <div style={{ display: 'flex', padding: '10px, 0px, 10px, 0px' }}>
```
Comment (dillydally414) [inline components/FormStyles/QuestionText.tsx:28]: any way we can make this a styled component for consistency hehe

File: components/FormStyles/RadioButton.tsx [hunk 16]
```diff
@@ -31,3 +29,33 @@ export const RadioButton = styled.input`
...
 `
+
+export const RadioInputIcon = styled.span`
+  border-radius: 50%;
+  width: 20px;
+  height: 20px;
+  box-shadow: inset 0 0 0 1px rgba(16, 22, 26, 0.2),
+    inset 0 -1px 0 rgba(16, 22, 26, 0.1);
+  background-color: #f5f8fa;
+  background-image: linear-gradient(
```
Comment (dillydally414) [inline components/FormStyles/RadioButton.tsx:40]: what does this linear-gradient do

File: components/FormStyles/RadioButton.tsx [hunk 17]
```diff
@@ -1,28 +1,26 @@
 import styled from 'styled-components'
+import { COLORS } from '../../constants/colors'
 
 export const RadioButton = styled.input`
```
Comment (dillydally414) [inline components/FormStyles/RadioButton.tsx:4]: Can you add a `:hover` rule to this as well bc for some reason when you hover over the button an ugly background appears:
Screen Shot 2022-04-20 at 9 03 56 PM

**Review by dillydally414 — CHANGES_REQUESTED** [review]
Looks pretty good just a few requested changes :-)

File: components/Critical/SideProgressBar.tsx [hunk 18]
```diff
@@ -1,100 +1,57 @@
 import React from 'react'
-import Box from '@material-ui/core/Box'
-import List from '@material-ui/core/List'
-import ListItem from '@material-ui/core/ListItem'
-import ListItemIcon from '@material-ui/core/ListItemIcon'
-import ListItemText from '@material-ui/core/ListItemText'
-import { withStyles } from '@material-ui/core'
+import {
+  withStyles,
```
Comment (clue4) [inline components/Critical/SideProgressBar.tsx:3]: thank you for cleaning up this janky stuff ;-; !!!
also seems like there might be a few unused imports on this file (and potentially other files haven't checked them yet)

File: components/FormStyles/Button.tsx [hunk 19]
```diff
@@ -1,23 +1,22 @@
 import styled, { css } from 'styled-components'
+import { COLORS } from '../../constants/colors'
```
Comment (clue4) [inline components/FormStyles/Button.tsx:2]: seems like this is also an unused import

**Review by clue4 — COMMENTED** [review]
iman you are a queen and this looks so beautiful (also ty designers!!!)

File: components/FormStyles/RadioButton.tsx (hunk 16 above)
Comment (imanmoreira) [reply components/FormStyles/RadioButton.tsx:40]: makes it pretty (short answer I have no idea but it makes it work nice, I stole it from the internet)

**Review by dillydally414 — APPROVED** [review]
it's okay. iguess

**Pull request merged**